frontend/composables/useAsyncEquipmentIds.ts

```diff
@@ -0,0 +1,53 @@
+import { isNonNullish, unique } from "remeda";
+
+import { GetOeeEquipmentTypesWithEquipmentIdsDocument } from "~/generated/graphql/operations";
+
+
+
+export default function () {
+  return useAsyncQuery(GetOeeEquipmentTypesWithEquipmentIdsDocument,
+    {},
+    "default",
+    { errorPolicy: "ignore" }
+    , {
+      transform: (typeRes) => {
```

Comment:
Because of the complexity of this function, let's extract it to a separate function in this file that we can pass to this object field.

Reply:
Removing the computeds has simplified it significantly. I think now, unless we're using it elsewhere, being in a composable is as broken out as it needs to be.